lib/resque/worker.rb

```diff
@@ -651,7 +651,7 @@ def unregister_worker(exception = nil)
       message = exception_while_unregistering.message
       if exception
         message = message + "\nOriginal Exception (#{exception.class}): #{exception.message}\n" +
-                            "  #{exception.backtrace.join("  \n")}"
+                            "  #{(exception.backtrace || []).join("  \n")}"
```

Comment:
Hmm interesting. Not sure I understand why this is necessary. Some exceptions don't have backtraces?